src/uri.rs

```diff
@@ -406,7 +407,8 @@ impl UUri {
     /// assert!(authority_a.is_remote_authority(&authority_b));
     /// ````
     pub fn is_remote_authority(&self, other_uri: &UUri) -> bool {
-        self.authority_name != WILDCARD_AUTHORITY
+        !self.authority_name.is_empty()
```

Comment:
a unit test would be nice, also covering the other cases ...

Reply:
Done